packages/bp/src/core/events/middleware-chain.test.ts

```diff
@@ -142,7 +142,8 @@ describe('Middleware', () => {
   })
 
   it('if should let mw set a custom timeout value', async () => {
-    const extraTime = 10
+    // Avoid a small number, bluebird overhead sometimes makes the test fail
+    const extraTime = 50
```

Comment:
Maybe we could crank this value up to 500ms to be extra safe?

Reply:
Setting this value higher slows down the test, the problem is mostly around 10ms. If we ever get this test fail again, we'll put higher, but it's still a nice to know what's the "threshold" of timeout delays